### ISSUE TITLE
package manager

### ISSUE THREAD

**Issue by oh-facts** [oh-facts/Eat-my-balls#1]
idk how to do this but what do i do so that cmakelists makes sure that you have sfml and if not, it installs it for you adds to path. Maybe does scoop install sfml or something depending on os. This way, I wont need a separate branch for linux since source files are exactly the same and only sfml dependencies are different. currently the cmake lists file is same for both programs and it checks if you have unix or win32 and accordingly links as it needs to. would be useful if dependencies could be installed this way too.

other option is that I use sparse git? submodule? source files are same, only vendor/SFML-2.5.1 changes depending on os.

tl;dr I dont want 2 branches, how do I have just 1 branch for both OS since source files are exactly the same, only dependencies are different (SFML in this case)
